mypy/dmypy_server.py

```diff
@@ -708,8 +708,15 @@ def find_added_suppressed(self,
             all_suppressed |= state.suppressed_set
 
         # Filter out things that shouldn't actually be considered suppressed.
+        #
+        # Various submodules of 'encodings' can be suppressed, since it uses
+        # module-level '__getattr__'. Skip them since there are many of them,
+        # and following imports to them is kind of pointless.
+        #
         # TODO: Figure out why these are treated as suppressed
-        all_suppressed = {module for module in all_suppressed if module not in graph}
+        all_suppressed = {module
+                          for module in all_suppressed
+                          if module not in graph and not module.startswith('encodings.')}
```

Comment:
Maybe make this a top level constant, so later we can skip similar "special" modules?

Reply:
I'll define a function that contains the check (as a micro-optimization and it also allows other checks than starts with).